Description: Disable internalAndroidPerformanceTesting
 No idea why this project keeps being run while being excluded, have to
 disable it by force.
Author: Kai-Chung Yan
Forwarded: not-needed
--- a/subprojects/performance/templates.gradle
+++ b/subprojects/performance/templates.gradle
@@ -339,13 +339,3 @@
     }
     templateArgs['projectDependencies'] = true
 }
-
-// The Android Studio performance tests use custom model classes which need to be found in
-// a dedicated project in order for the classpath to be setup properly
-def androidRemoteProjects = tasks.withType(RemoteProject).matching { it.name =~ /Android/ }
-androidRemoteProjects.all {
-    dependsOn tasks.findByPath(':internalAndroidPerformanceTesting:buildClassPath')
-    doLast {
-        file("$outputDirectory/tapi-classpath.txt") << project(':internalAndroidPerformanceTesting').buildClassPath.outputFile.text
-    }
-}
--- a/settings.gradle
+++ b/settings.gradle
@@ -51,7 +51,6 @@
 include 'internalTesting'
 include 'internalIntegTesting'
 include 'internalPerformanceTesting'
-include 'internalAndroidPerformanceTesting'
 include 'performance'
 include 'buildScanPerformance'
 include 'javascript'
